docs/DeveloperGuide.md

```diff
@@ -630,7 +630,7 @@ The `EditEventCommand` class is responsible for editing a specific Event for a p
   `DATE_OR_DATETIME_OF_EVENT` parameter.
 * With `TIME`, an example of the `DATE_OR_DATETIME_OF_EVENT` is `20-12-2025, 12:00 - 15:00`
 * Without `TIME`, an example of the `DATE_OR_DATETIME_OF_EVENT` is `20-12-2025`
-* EditEventCommand will edit the selected `EVENT_INDEX` with a new event.
+* EditEventCommand will edit the selected `EVENT_INDEX` with a new event for the selected `PATIENT_INDEX`
```

Comment:
Would it be better to have full stop after `PATIENT_INDEX ` to standardise across the DG?

Reply:
Yes, I agree!